src/components/Composer/index.js

```diff
@@ -407,7 +338,19 @@ class Composer extends React.Component {
 Composer.propTypes = propTypes;
 Composer.defaultProps = defaultProps;
 
+const ComposerWithDragAndDrop = props => (
+    <DragAndDrop onDragOver={props.onDragOver} onDragLeave={props.onDragLeave} onDragEnter={props.onDragEnter} onDrop={props.onDrop}>
```

Comment:
Let's move this in the ReportScreen. I don't any relation on DargAndDrop with Composer.